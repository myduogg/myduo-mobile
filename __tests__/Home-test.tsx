import React from 'react';
import {create} from 'react-test-renderer';
import Home from '../src/scenes/home';

const home = create(<Home />);

it('home snapshot', () => {
  expect(home).toMatchSnapshot();
});
