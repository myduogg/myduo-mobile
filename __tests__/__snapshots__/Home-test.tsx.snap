// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home snapshot 1`] = `
<View>
  <Text>
    Estou em home!
  </Text>
</View>
`;
